src/ensembl/src/services/storage-service.ts

```diff
@@ -2,19 +2,17 @@ import windowService, {
   WindowServiceInterface
 } from 'src/services/window-service';
 import merge from 'lodash/merge';
+import JSONValue from 'src/shared/types/JSON';
 
 export enum StorageType {
   LOCAL_STORAGE = 'localstorage',
   SESSION_STORAGE = 'sessionstorage'
 }
 
 type PrimitiveValue = string | number | boolean | null | undefined;
-type ArrayValue = PrimitiveValue[] | ObjectValue[];
-type ObjectValue = {
-  [key: string]: PrimitiveValue | ArrayValue | ObjectValue;
-};
+type ArrayValue = PrimitiveValue[] | JSONValue[];
```

Comment:
Could you import PrimitiveValue and ArrayValue from `types/JSON.ts` as well?

Reply:
That's done